### PR TITLE
zh-trans: add /docs/reference/setup-tools/kubefed/kubefed-init.md

### PR DESCRIPTION
**Pull request by AdamDang**
zh-trans: add /docs/reference/setup-tools/kubefed/kubefed-init.md

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+      --image-pull-policy string                 PullPolicy describes a policy for if/when to pull a container image. The default pull policy is IfNotPresent which will not pull an image if it already exists. (default "IfNotPresent")
+      --image-pull-secrets string                Provide secrets that can access the private registry.
+      --node-selector string                     comma separated list of nodeSelector arguments: Example "arg1=value1,arg2=value2..."
+```
+-->
+
+```
+      --api-server-advertise-address string      公布 API 服务器的 nodeport 服务的首选地址。只在 'api-server-service-type=NodePort' 时有效。
+      --api-server-port int32                    API 服务器的 nodeport 服务的首选端口(设置为 0 将随机指定端口)。只在 'api-server-service-type=NodePort' 时有效。
+      --api-server-service-type string           创建联邦 API 服务器的服务类型。选项包括：'LoadBalancer' (默认)，'NodePort'。(默认 "LoadBalancer")
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md]: OK，主要是扩号的处理。

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+      --apiserver-arg-overrides string           要重写的联合 API 服务器参数列表，用逗号分隔，例如 "--arg1=value1,--arg2=value2..."
+      --apiserver-enable-basic-auth              为联邦 API 服务器启用 HTTP 基本身份验证。默认关闭。
+      --apiserver-enable-token-auth              为联邦 API 服务器启用令牌身份验证。默认关闭。
+      --controllermanager-arg-overrides string   要重写的 federation-controller-manager 参数列表，用逗号分隔，例如："--arg1=value1,--arg2=value2..."
+      --credentials-kubeconfig string            本地文件系统上的 kubeconfig 文件路径，应用于与主机群集或加入群集（而不是默认的 kubeconfig）进行身份验证。这可用于在初始化联邦控制平面或将群集加入到一个群集时覆盖基于 RBAC 的身份验证，即使在群集公开 RBAC API 时也是如此。
+      --dns-provider string                      此 deployment 使用的 DNS 驱动。
+      --dns-provider-config string               用来配置 DNS 驱动的本地文件系统中的配置文件。
+      --dns-zone-name string                     此联邦的 DNS 后缀。使用此后缀发布联邦服务 DNS 名称。
+      --dry-run                                  在不向服务器发送命令的情况下进行试运行。
+      --etcd-image string                        服务器使用的镜像（默认 "gcr.io/google_containers/etcd:3.1.10"）
```
Comment (chenrui333) [inline content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md]:
```suggestion
      --etcd-image string                        服务器使用的镜像（默认值为 "gcr.io/google_containers/etcd:3.1.10"）
```

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+      --apiserver-enable-basic-auth              Enables HTTP Basic authentication for the federation-apiserver. Defaults to false.
+      --apiserver-enable-token-auth              Enables token authentication for the federation-apiserver. Defaults to false.
+      --controllermanager-arg-overrides string   comma separated list of federation-controller-manager arguments to override: Example "--arg1=value1,--arg2=value2..."
+      --dns-provider string                      Dns provider to be used for this deployment.
+      --dns-provider-config string               Config file path on local file system for configuring DNS provider.
+      --dns-zone-name string                     DNS suffix for this federation. Federated Service DNS names are published with this suffix.
+      --dry-run                                  dry run without sending commands to server.
+      --etcd-image string                        Image to use for etcd server. (default "gcr.io/google_containers/etcd:3.1.10")
+      --etcd-persistent-storage                  Use persistent volume for etcd. Defaults to 'true'. (default true)
```
Comment (chenrui333) [inline content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md]: 这个原文感觉有重复。

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+      --apiserver-enable-basic-auth              为联邦 API 服务器启用 HTTP 基本身份验证。默认关闭。
+      --apiserver-enable-token-auth              为联邦 API 服务器启用令牌身份验证。默认关闭。
+      --controllermanager-arg-overrides string   要重写的 federation-controller-manager 参数列表，用逗号分隔，例如："--arg1=value1,--arg2=value2..."
+      --credentials-kubeconfig string            本地文件系统上的 kubeconfig 文件路径，应用于与主机群集或加入群集（而不是默认的 kubeconfig）进行身份验证。这可用于在初始化联邦控制平面或将群集加入到一个群集时覆盖基于 RBAC 的身份验证，即使在群集公开 RBAC API 时也是如此。
+      --dns-provider string                      此 deployment 使用的 DNS 驱动。
+      --dns-provider-config string               用来配置 DNS 驱动的本地文件系统中的配置文件。
+      --dns-zone-name string                     此联邦的 DNS 后缀。使用此后缀发布联邦服务 DNS 名称。
+      --dry-run                                  在不向服务器发送命令的情况下进行试运行。
+      --etcd-image string                        服务器使用的镜像（默认 "gcr.io/google_containers/etcd:3.1.10"）
+      --etcd-persistent-storage                  etcd 使用持久卷，默认为 'true'。（默认 true）
```
Comment (chenrui333) [inline content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md]:
```suggestion
      --etcd-persistent-storage                  etcd 使用持久卷，默认为 'true'。（默认值为 true）
```

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+      --etcd-pv-capacity string                  etcd 所使用的持久卷申领的大小（默认 "10Gi"）
```
Comment (chenrui333) [inline content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md]:
```suggestion
      --etcd-pv-capacity string                  etcd 所使用的持久卷申领的大小（默认值为 "10Gi"）
```

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+      --etcd-pv-storage-class string             etcd 所使用的持久卷申领的存储类。如果主机集群没有配置默认存储类，就必须提供。
+      --etcd-servers string                      用来存储联邦状态的外部预部署的 etcd 服务器。
+      --federation-system-namespace string       主机集群上用来安装联邦系统组件的命名空间（默认 "federation-system"）
+  -h, --help                                     init 的帮助信息
+      --host-cluster-context string              主机集群上下文
+      --image string                             用于联邦 API 服务器和控制器管理器二进制文件的镜像。（默认 "gcr.io/k8s-jkns-e2e-gce-federation/fcp-amd64:v0.0.0-master_$Format:%h$"）
```
Comment (chenrui333) [inline content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md]:
```suggestion
      --image string                             用于联邦 API 服务器和控制器管理器二进制文件的镜像。（默认值为 "gcr.io/k8s-jkns-e2e-gce-federation/fcp-amd64:v0.0.0-master_$Format:%h$"）
```

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+      --image-pull-policy string                 拉取策略描述了容器映像的拉取策略。默认的拉取策略是 ifNotPresent，如果镜像已存在，则不会拉它。(默认 "IfNotPresent")
```
Comment (chenrui333) [inline content/zh/docs/reference/setup-tools/kubefed/kubefed-init.md]:
```suggestion
      --image-pull-policy string                 拉取策略描述了容器映像的拉取策略。默认的拉取策略是 ifNotPresent，如果镜像已存在，则不会拉它。(默认值为 "IfNotPresent")
```

**Comment by chenrui333** [comment]
Putting this PR on hold in favor of the newer one.
/hold

**Comment by chenrui333** [comment]
@AdamDang Let's merge these discussions into another PR. 😄 

/close

**Pull request closed without merging**